event-subscription/lambda/README.md

```diff
@@ -94,6 +94,18 @@ kubeless get-server-config
         ```bash
         kubectl logs -f $(kubectl get po -n $KYMA_EXAMPLE_NS -l function=hello-with-data --no-headers | grep -i running | awk '{print $1}') -c hello-with-data -n $KYMA_EXAMPLE_NS
         ```
+6. To access attributes of events which are compliant to CloudEvents [specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md), you can access event object as follows:

```

Comment:
```suggestion
6. To access Event attributes compliant with the CloudEvents [specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md) use the following paths in the request:
```
Question: can we write "paths" here? How are the attributes accessed - is it a specific request which uses the path?

Reply:
Yes we can and with more context.
`use the following paths for the event object in the request:`
Will it make sense?